proposed/2021/NuGet.orgTFMs.md

```diff
@@ -0,0 +1,107 @@
+- .NET Framework (net) 
+- Xamarin/Mono (monoandroid, monotouch, xamarinios, monomac, xamarinmac, xamarinwatchos, xamarintvos) 
+- .NET Core (netcoreapp) 
+- .NET Standard (netstandard) 
+- .NET 5 (net5.0) 
+
+### Technical Explanation
+- Packages whose layout follow our [guidelines](https://docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks) will be examined when uploaded and validated, prior to listing. Target Frameworks will be determined and displayed here.
+- Note that packages which do not follow the guidelines may have inconsistent results.
+- In cases where versions are not specified (such as .net) or other unexpected scenarios, these will be considered v0.0 and will be displayed as such. 
+- When a portable TFM is being broken up and displayed (if feasible), we will need to flag it as portable, as portable assets are not the same as platform assets.
+
+## Drawbacks
+- We would need to focus on compatibility with PackageReference and not packages.config projects, as these two support compatibility in different ways.
+
+## Rationale and alternatives
+Q: I think build/{packageid}.[props|targets] poses a problem, because from NuGet's point of view this package is compatible with all TFMs, .NET, native/c++, or anything else. For packages like Nerdbank.GitVersioning, or SourceLink, this makes sense. But many .NET packages incorrectly use this, rather than build\{tfm}\{packageid}.[props|targets]. I think it's worthwhile pointing this out and explaining what our plan is for these packages.
+A: Let's consider putting a link to a document which can call out corner cases such as this, and respond to direct customer concerns with advice on nupkg construction conducive to correct supprted TFM assessment.
+
+Q: Will the "Frameworks" section be expanded by default? Should the "Frameworks" section explain how the customer should use this information? Should we add a link to docs? Consider moving up the "Frameworks" section. I would suggest "Documentation" first, followed by "Frameworks". The "Version History" table is noisy and it can be difficult to see what's below it. The "Dependencies" and "Used By" sections are useful information, but would claim "Frameworks" is more important.
+A: Let's not expand by default--this will give us the option of trackiong the popularity of the section by tracking expansion clicks.
+
+Q: Potentials for modelling our UX on:
+ - https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder?view=net-5.0#applies-to
+ - https://apisof.net/catalog/e40ef6ad-bd6e-9f29-7864-b942e37be11f
+ - https://docs.microsoft.com/en-us/dotnet/standard/frameworks#latest-versions
+A: From a sample of 100,000 package/versions, we have:
+ - Only 445 have more than 5 supported TFMs
+ - Only 16 have more than 10
+ - Only 1 more than 15 (and it has 16)
+ So, we'll go with a flat list for now and look at other options for subsequent work if desired.
```

Comment:
I can also work with folks on the UX side, accessibility of information here is more of a PM/UX question so happy to chase that down